hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/ha/RequestHedgingProxyProvider.java

```diff
@@ -234,8 +234,6 @@ private void logProxyException(Exception ex, String proxyInfo) {
         LOG.debug("Invocation returned standby exception on [" +
             proxyInfo + "]");
       }
-    } else {
-      LOG.warn("Invocation returned exception on [" + proxyInfo + "]");
```

Comment:
why removing this message at least keep in debug , unless you are sure that when you reach this step you have already logged this exception elsewhere

Reply:
The logs are duplicated, they'll be shown at the level of RetryInvocationHandler caller.